### DISCUSSION TITLE
2024-06-15(토)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#147] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by devwqc** [comment]
### 완료한 일
- BottomSheet 완성
- CartButton 장바구니 수량 작업

### 해야할 일
- BottomSheet, Modal 소스 다듬고 PR 올리기
- 페이지 작업 시작하기

### 어려운 일
- 로그아웃 직후에 CartButton의 장바구니 수량이 새로고침 해야지 0으로 표시되는 현상

**Comment by Seoin02** [comment]
### 완료한 일
- 인증 코드리뷰 반영

### 해야할 일
- 맞춤 상품, 주문 상세 페이지 컴포넌트 분리
- 온보딩, 프로필 API 연결
- #146 

### 어려운 일
- 인증 인가 작업을 마무리했으나 이슈들이 좀 있는 것 같습니다. 다들 가입 한 번 해보시고, 수정할 부분 알려주시면 감사하겠습니다!

**Comment by minyoung0503** [comment]
### 완료한 일
상세 페이지 UI

### 해야할 일
상품 데이터 컴포넌트 분리 후 PR

### 어려운 일

**Comment by q45402sk** [comment]
### 완료한 일

### 해야할 일
상품상세- 비슷한 상품 스크롤 적용
myDeliveryPage
purchaseDeliveryPage
purchaseDeliveryEditPage

### 어려운 일
